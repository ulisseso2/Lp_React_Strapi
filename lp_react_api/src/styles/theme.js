export const theme = {
  colors: {
    primary: '#457B9D',
    light: '#A8DADC',
    dark: '#1D3557',
    hot: '#E63946',
    white: '#f7fff7',
    black: '#343434',
  },
  fonts: {
    family: {
      main: "'Open Sans', sans-serif",
      title: "'Montserrat', sans-serif",
      body: "'Open Sans', sans-serif",
    },
  },
};
//tema padrão da aplicação usado em vários componentes
