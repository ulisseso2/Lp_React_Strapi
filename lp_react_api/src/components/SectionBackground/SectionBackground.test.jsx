import { screen } from '@testing-library/react';
import { SectionBackground } from '.';
import { renderTheme } from '../../styles/render.theme';

describe('<SectionBackground>', () => {
  it('should render Background', () => {
    const { container } = renderTheme(
      <SectionBackground background={true}>
        <h1>children</h1>
      </SectionBackground>,
    );
    expect(screen.getByRole('heading')).toBeInTheDocument();
    expect(container).toMatchSnapshot();
  });
});
