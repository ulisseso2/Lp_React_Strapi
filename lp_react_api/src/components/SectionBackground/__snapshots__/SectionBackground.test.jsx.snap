// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SectionBackground> should render Background 1`] = `
<div>
  <div
    class="sc-bczRLJ lqupo"
  >
    <h1>
      children
    </h1>
  </div>
   
</div>
`;
